### PR TITLE
Only use tb_writer from master

### PR DESCRIPTION
**Pull request by jysohn23**

### CODE REVIEW

File: examples/run_glue_tpu.py [hunk 1]
```diff
@@ -177,15 +178,17 @@ def train(args, train_dataset, model, tokenizer, disable_logging=False):
...
                         results = evaluate(args, model, tokenizer, disable_logging=disable_logging)
-                        for key, value in results.items():
-                            tb_writer.add_scalar('eval_{}'.format(key), value, global_step)
                     loss_scalar = loss.item()
-                    tb_writer.add_scalar('lr', scheduler.get_lr()[0], global_step)
-                    tb_writer.add_scalar('loss', loss_scalar, global_step)
                     logger.info('global_step: {global_step}, lr: {lr:.3f}, loss: {loss:.3f}'.format(
                         global_step=global_step, lr=scheduler.get_lr()[0], loss=loss_scalar))
+                    if xm.is_master_ordinal():
+                        for key, value in results.items():
```
Comment (taylanbil) [inline examples/run_glue_tpu.py:189]: similar comment to prev. pr; all the values here need to be on cpu. if so, can you add a comment? it's a subtle point.

**Review by taylanbil — APPROVED** [review]
why not log training related info? only eval?

File: examples/run_glue_tpu.py (hunk 1 above)
Comment (jysohn23) [reply examples/run_glue_tpu.py:189]: Done, thanks.

**Comment by jysohn23** [comment]
> why not log training related info? only eval?

What do you mean by we don't log training related info? We do `logger.info` both in the `train` and `evaluate` methods with metrics.

**Comment by taylanbil** [comment]
I mean log to tensorboard during training. not a big deal.

**Comment by jysohn23** [comment]
> I mean log to tensorboard during training. not a big deal.

Oh I see. We actually do log training loss and lr in tensorboard during training.
